tsconfig.json

```diff
@@ -18,6 +18,8 @@
     "moduleResolution": "node",
     "jsx": "preserve",
     "baseUrl": ".",
+    "outDir": "builds",
+    "incremental": true,
```

Comment:
Next now automatically adds this when you start the app, so we need to have an `outDir` and modify `.gitignore`. See more details about the combination of new and existing config options [here](https://github.com/microsoft/TypeScript/issues/30661).

Reply:
ok